workers/unity/.gitignore

```diff
@@ -23,3 +23,6 @@
 # Jetbrains
 /Assets/Plugins/Editor/Jetbrains
 /Assets/Plugins/Editor/Jetbrains.meta
+
+# Sonarcube dependency
+global.json
```

Comment:
This shouldn't be in the gitignore, and the file won't ever survive a `test.sh` run

Reply:
Why was this resolved?

Reply:
I think I clicked it by mistake when trying to resolve one of the other 2, anyway, I fixed it now